prepare_source

```diff
@@ -1 +1,3 @@
-git_src --branch debian/2.12.7+dfsg-3 https://salsa.debian.org/xml-sgml-team/libxml2.git
```

Comment:
downgrading from 2.12.7 to 2.9.14 on the main branch does not seem like a good idea.

I think what you should do is to create a rel-1443 branch and do the pr to that branch.

question: are libxml versions in 1592 and today also affected, or do you want to do this only for 1443?

Reply:
Technically for the repo a separate branch is not required but we do it in all other repos for a good reason, so I'd like to have this here the same. Avoid downgrade on `today`.

Reply:
I've tried building this via a separated branch, but the release workflow will only work for the `main` branch, as it seems. Since we can upgrade `libxml2` to a more recent version afterward, I think it's fine.